### DISCUSSION TITLE
Raspberry pi with user in group render gives a blank screen

### DISCUSSION THREAD

**Discussion by beetlegigg** [neutrinolabs/xrdp#2271] [Q&A]
With the latest rpi os 64 bit on a pi4 (4gb) and mac os (big sur) with ms remote desktop app, connecting with remote desktop (RD) and a 'pi' user just gives a blank screen. 
However creating a new user and connecting through RD then the rpi os desktop comes up ok.  

But I then added the new user to the same groups as pi belonged to and again just a blank screen.  Trial and error showed that removing the new user from the render group enabled the pi desktop to work.

I have not removed the render group from pi to see if then pi would work with RD for reasons of timidity.
Can a user in a render group be make to work ok ?

**Comment by matt335672** [comment]
This is a known issue with the pi kernel.

See issue #2060 for a detailed analysis (in particular [this post](https://github.com/neutrinolabs/xrdp/issues/2060#issuecomment-979958020)), and also discussion #2129.
